proto/enums/enums.proto

```diff
@@ -408,13 +412,25 @@ enum StatusCode {
     CONFIG_PROFILES_NOT_FOUND_ERR = 1212;
     // Config API version not found
     CONFIG_UNKNOWN_API_VERSION_ERR = 1213;
+    // The validator is not supported in skaffold-managed mode (not whitelisted).
+    CONFIG_UNKNOWN_VALIDATOR = 1214;
+    // The mutator is not supported in skaffold-managed mode (not whitelisted).
+    CONFIG_UNKNOWN_MUTATOR = 1215;
 
     // Inspect command errors
 
     // Catch-all `skaffold inspect` command error
     INSPECT_UNKNOWN_ERR = 1301;
     // Trying to add new build environment that already exists
     INSPECT_BUILD_ENV_ALREADY_EXISTS_ERR = 1302;
+
+    // Kptfile errors
+    // The Kptfile is not a valid yaml file
+    KPTFILE_INVALID_YAML_ERR = 1401;
+    // The Kptfile is not a valid API schema
+    KPTFILE_INVALID_SCHEMA_ERR = 1402;
+    // The Kptfile cannot be created via `kpt pkg init` or `kpt live init`.
+    KPTFILE_INIT_ERR = 1403;
```

Comment:
Prefix with `Deploy` as suggested in the file here https://github.com/GoogleContainerTools/skaffold/blob/480bcebbd305cf6d6c49bab8b3a65342f9aca7d8/proto/enums/enums.proto#L89